code/modules/paperwork/paper.dm

```diff
@@ -437,3 +437,83 @@
 
 /obj/item/paper/crumpled/bloody
 	icon_state = "scrap_bloodied"
+
+//Evil faxes
+/obj/item/paper/evilfax
+	name = "CentCom Reply"
+	info = ""
+	var/mytarget = null
+	var/myeffect = null
+	var/used = FALSE
+	var/countdown = 60
+	var/activate_on_timeout = FALSE
+
+/obj/item/paper/evilfax/examine(var/mob/user, var/forceshow = FALSE, var/forcestars = FALSE, var/infolinks = FALSE, var/view = TRUE)
+	if(user == mytarget)
+		if(istype(user, /mob/living/carbon))
+			var/mob/living/carbon/C = user
+			evilpaper_specialaction(C)
+			..()
```

Comment:
```suggestion
			. = ..()
```
I'm not sure about this codebase, but it runtimes on mine.

Reply:
Aye, I'm sure this is because of the examine code refactor we merged a while ago.